src/main/java/org/ieeervce/gatekeeper/controller/RoleController.java

```diff
@@ -0,0 +1,44 @@
+package org.ieeervce.gatekeeper.controller;
+
+import org.ieeervce.gatekeeper.ItemNotFoundException;
+import org.ieeervce.gatekeeper.dto.RoleDTO;
+import org.ieeervce.gatekeeper.dto.SocietyDTO;
+import org.ieeervce.gatekeeper.entity.Role;
+import org.ieeervce.gatekeeper.service.RoleService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/role")
+public class RoleController {
+    private final RoleService roleService;
+    private final ModelMapper modelMapper;
+    @Autowired
+    public RoleController(RoleService roleService, ModelMapper modelMapper){
+        this.roleService= roleService;
+        this.modelMapper=modelMapper;
+    }
+    @GetMapping("/{roleId}")
```

Comment:
Please consider implementing a get all roles route, we need that have a list of roles a user can take when they're customizing.